2226/2/table_printer.rb

```diff
@@ -0,0 +1,21 @@
+# This class is used for printing table to the terminal
+class TablePrinter
```

Comment:
This class still contains a lot of logic program logic but it's name is just table printer. Does it print HTML tables code?